sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/client.py

```diff
@@ -342,7 +351,7 @@ def import_certificate(
         """Imports a certificate into a specified key vault.
 
         Imports an existing valid certificate, containing a private key, into
-        Azure Key Vault. The certificate to be imported can be in either PFX or
+        Azure Key Vault. The certificate to be imported can be in either PFX orHi
```

Comment:
Pretty sure you don't intend this change 😁

Reply:
true, thanks!